public/components/layout.js

```diff
@@ -8,529 +8,542 @@
-        <a style="margin: 0 10px;" class="active" href="about.html"> About Geolocator </a>
-        <a style="margin: 0 10px;" class="active" href="moreLinks.html"> Additional Links </a>
+	<a style="margin: 0 10px;" class="active" href="index.html"> Home </a>
+	<a style="margin: 0 10px;" class="active" href="about.html"> About Geolocator </a>
+	<a style="margin: 0 10px;" class="active" href="moreLinks.html"> Additional Links </a>
     </nav>
     `
-    constructor(){
-        super()
-        this.attachShadow({mode:'open'})
-        this.shadowRoot.innerHTML = this.#navTmpl
-    }
+	constructor(){
+		super()
+		this.attachShadow({mode:'open'})
+		this.shadowRoot.innerHTML = this.#navTmpl
+	}
 }
 
 customElements.define("geo-nav",GeoNav)
 
 class GeoFooter extends HTMLElement {
-    #footerTmpl = `
+	#footerTmpl = `
     <link rel="stylesheet" href="https://unpkg.com/chota@latest">
     <link rel="stylesheet" href="stylesheets/style.css">
     <footer class="nav nav-center text-primary is-fixed is-full-width is-vertical-align">
-        <a target="_blank"
-            href="https://www.slu.edu/research/faculty-resources/research-computing.php"><small>&copy;2022 Research
-                Computing Group</small></a>
-        <a target="_blank" href="https://slu.edu"><img class="brand" alt="SLU logo"
-                src="https://www.slu.edu/marcom/tools-downloads/imgs/logo/left-aligned/slu_logoleftaligned_rgb.png"></a>
-        <a target="_blank" href="https://www.slu.edu/arts-and-sciences/ong-center/"><img class="brand"
-                alt="SLU RCG logo"
-                src="https://centerfordigitalhumanities.github.io/media-assets/logos/rcg-logo.png"></a>
-        <a target="_blank" href="http://rerum.io"><img class="brand" alt="Rerum logo"
-                src="https://centerfordigitalhumanities.github.io/blog/assets/images/rerum.jpg"><small>RERUM
-                v1</small></a>
-	    <a target="_blank" href="https://github.com/CenterForDigitalHumanities/geolocator.git"><img width="30" height="30" alt="Git logo" src="./github.svg">
-        <small>&nbsp;&nbsp;Support/Contribute to Geolocator</small></a>
+
+	<a target="_blank"
+		href="https://www.slu.edu/research/faculty-resources/research-computing.php">
+		<small class = "tooltiptext">&copy;2022 Research Computing Group</small></a>
+
+	<a target="_blank" class="tooltip" href="https://slu.edu">
+		<img class="brand" alt="SLU logo" src="https://www.slu.edu/marcom/tools-downloads/imgs/logo/left-aligned/slu_logoleftaligned_rgb.png">
+		<span class= "tooltiptext">SLU Homepage</span></a>
+
+	<a target="_blank" class = "tooltip" id="slu" href="https://www.slu.edu/arts-and-sciences/ong-center/">
+		<img class="brand" alt="SLU RCG logo" src="https://centerfordigitalhumanities.github.io/media-assets/logos/rcg-logo.png">
+		<span class= "tooltiptext">SLU Research Group</span></a>
+
+
+	<a target="_blank" class= "tooltip"id = "rerum" href="http://rerum.io">
```

Comment:
Also not your fault.  Finally got our act together and have https

`href="https://rerum.io"`